Task 10/modules/numbers/numbers.js

```diff
@@ -1,5 +1,5 @@
-let one = 1;
-let two = 2;
-let three = 3;
-let four = 4;
-let five = 5;
+export let one = 1;
```

Comment:
Buvo galima pataisyti visur į const, nes let reikėtų vengti, nebent tikrai norėsim leisti vertes perrašinėti.